stable/sysdig/README.md

```diff
@@ -176,6 +176,32 @@ And then, upgrade Helm chart with:
 $ helm upgrade my-release -f values.yaml stable/sysdig
 ```
 
+## How to upgrade to the last version
+
+In case you deployed the chart with a values.yaml file, you just need to modify (or add if it's missing) the `agent.tag` field and execute:
+
+```bash
+$ helm install --name my-release -f values.yaml stable/sysdig
+```
+
+If you deployed the chart setting the values as CLI parameters, like for example:
```

Comment:
you say --name my-release and then --name sysdig which dont match.

can you explain if we need to use the same or different?